bin/qds.py

```diff
@@ -524,6 +542,10 @@ def main():
                          default=False,
                          help="skip verification of server SSL certificate. Insecure: use with caution.")
 
+    optparser.add_option("--cloud", dest="cloud",
+                         default=os.getenv('CLOUD_PROVIDER'),
+                         help="cloud", choices=["AWS", "AZURE", "ORACLE_BMC"])
```

Comment:
Specifying it inside qubole.py is fine. We should also set the default here as well just like we are setting defaults for `api_token`, `api_url` etc